src/acceptance/java/io/codurance/pouch/DatabaseHelper.java

```diff
@@ -37,6 +38,14 @@ public String getTitle() {
         public String getUrl() {
             return url;
         }
+
+        public boolean isFavourite() {
+            return favourite;
+        }
+
+        public void setFavourite(boolean favouriteStatus) {
```

Comment:
I would remove the parameter here and add a new method removeFavourite, but that's only my preference

Reply:
I get it, makes sense.